src/phoenix/server/api/routers/v1/datasets.py

```diff
@@ -615,31 +617,192 @@ async def get_dataset_csv(request: Request) -> Response:
+    tags:
+      - datasets
+    parameters:
+      - in: path
+        name: id
+        required: true
+        schema:
+          type: string
+        description: Dataset ID
+      - in: query
+        name: version
+        schema:
+          type: string
+        description: Dataset version ID. If omitted, returns the latest version.
+    responses:
+      200:
+        description: Success
+        content:
+          text/plain:
+            schema:
+              type: string
+              contentMediaType: text/plain
+              contentEncoding: gzip
+      404:
+        description: Dataset does not exist.
+      422:
+        description: Dataset ID or version ID is invalid.
+    """
+    try:
+        dataset_name, examples = await _get_db_examples(request)
+    except ValueError as e:
+        return Response(content=str(e), status_code=HTTP_422_UNPROCESSABLE_ENTITY)
+    content = await run_in_threadpool(_get_content_jsonl_openai_ft, examples)
+    return Response(
+        content=content,
+        headers={
+            "content-disposition": f'attachment; filename="{dataset_name}.jsonl"',
+            "content-type": "text/plain",
+            "content-encoding": "gzip",
+        },
+    )
+
+
+async def get_dataset_jsonl_openai_evals(request: Request) -> Response:
+    """
+    summary: Download dataset examples as OpenAI Evals JSONL file
+    operationId: getDatasetJSONLOpenAIEvals
+    tags:
+      - datasets
+    parameters:
+      - in: path
+        name: id
+        required: true
+        schema:
+          type: string
+        description: Dataset ID
+      - in: query
+        name: version
+        schema:
+          type: string
+        description: Dataset version ID. If omitted, returns the latest version.
+    responses:
+      200:
+        description: Success
+        content:
+          text/plain:
+            schema:
+              type: string
+              contentMediaType: text/plain
+              contentEncoding: gzip
+      404:
+        description: Dataset does not exist.
+      422:
+        description: Dataset ID or version ID is invalid.
+    """
+    try:
+        dataset_name, examples = await _get_db_examples(request)
+    except ValueError as e:
+        return Response(content=str(e), status_code=HTTP_422_UNPROCESSABLE_ENTITY)
+    content = await run_in_threadpool(_get_content_jsonl_openai_evals, examples)
+    return Response(
+        content=content,
+        headers={
+            "content-disposition": f'attachment; filename="{dataset_name}.jsonl"',
+            "content-type": "text/plain",
+            "content-encoding": "gzip",
+        },
+    )
+
+
+DatasetName: TypeAlias = str
+ExampleId: TypeAlias = int
+ExampleInput: TypeAlias = Mapping[str, Any]
+ExampleOutput: TypeAlias = Mapping[str, Any]
+ExampleMetadata: TypeAlias = Mapping[str, Any]
+
+DBExamples: TypeAlias = List[Tuple[ExampleId, ExampleInput, ExampleOutput, ExampleMetadata]]
```

Comment:
Suggestion: This feels like it could just be `models.DatasetExampleRevision` if we did

```
        select(
            mder
        )
```

instead of

```
        select(
            mder.dataset_example_id,
            mder.input,
            mder.output,
            mder.metadata_,
        )
```

Reply:
ok

Reply:
updated. thx